Modules/Loadable/Data/SubjectHierarchyPlugins/DataSubjectHierarchyPlugin.py

```diff
@@ -0,0 +1,147 @@
+import logging
+import vtk, qt, ctk, slicer
+from slicer.ScriptedLoadableModule import *
+
+from SubjectHierarchyPlugins import AbstractScriptedSubjectHierarchyPlugin
+
+class DataSubjectHierarchyPlugin(AbstractScriptedSubjectHierarchyPlugin):
+  """ Scripted subject hierarchy plugin for the ExportAs module.
+  """
+
+  # Necessary static member to be able to set python source to scripted subject hierarchy plugin
+  filePath = __file__
+
+  def __init__(self, scriptedPlugin):
+    AbstractScriptedSubjectHierarchyPlugin.__init__(self, scriptedPlugin)
+
+    pluginHandlerSingleton = slicer.qSlicerSubjectHierarchyPluginHandler.instance()
+    self.subjectHierarchyNode = pluginHandlerSingleton.subjectHierarchyNode()
+
+    self.exportAsAction = qt.QAction(f"Export as...", scriptedPlugin)
+    self.exportTransformedAsAction = qt.QAction(f"Export transformed as...", scriptedPlugin)
+    self.menu = qt.QMenu("Plugin Menu")
```

Comment:
In Subject hierarchy basically all plugins use sentence case. If that's not a big deal please change these.